src/UI/controllers/HTMLTooltipUIController.js

```diff
@@ -328,23 +328,25 @@ export class HTMLTooltipUIController extends UIController {
 
     /**
      * Called when clicking on the Manage… button in the html tooltip
-     *
      * @param {SupportedMainTypes} type
      * @returns {*}
      * @private
      */
     _onManage(type) {
-        this.removeTooltip();
         switch (type) {
             case 'credentials':
-                return this._options.device.openManagePasswords();
+                this._options.device.openManagePasswords();
+                break;
             case 'creditCards':
-                return this._options.device.openManageCreditCards();
+                this._options.device.openManageCreditCards();
+                break;
             case 'identities':
-                return this._options.device.openManageIdentities();
+                this._options.device.openManageIdentities();
+                break;
             default:
             // noop
         }
+        this.removeTooltip();
```

Comment:
Doesn't seem to affect MacOS - would like to avoid multiple/split implementation between platforms unless really necessary (doesn't seem like it).